lib/definition.js

```diff
@@ -354,18 +343,18 @@ module.exports = class Definition {
 
   _validateList() {
     assert(this._list_type, `Invalid Env Config (${this._name}): Missing .type()`)
-    const is_array = Array.isArray(this._value)
-    if (this._required && (!is_array || !this._value.length)) {
+    if (!Array.isArray(this._value)) return
+
+    if (this._required && !this._value.length) {
       throw new MissingEnvError({
-        type: 'list'
+        type: this._type
```

Comment:
I'm not sure this is accurate. its a list of type

Reply:
Are you saying to hardcode the type to `list` again? I just pasted this code, so that'd be fine.

Reply:
I guess I'm asking. Does the error still say something that is helpful / accurate ? I'm not sure what the type value is when it's a list

Reply:
I think it's kind of confusing either way. The error should probably say `[]${type}`